app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -410,38 +383,33 @@ class AttendeeFragment : Fragment() {
             }
             val country = if (country.text.isEmpty()) country.text.toString() else null
             attendeeViewModel.createAttendees(attendees, country, selectedPaymentOption)
+
+            attendeeViewModel.isAttendeeCreated.observe(this, Observer { isAttendeeCreated ->
+                if (isAttendeeCreated) {
+                    if (selectedPaymentOption == getString(R.string.stripe))
```

Comment:
I guess these two if statements can be combined

Reply:
yes, but paypal option is also there. Again need to separate conditions in next issue. Should I combine it for now?

Reply:
Updated

Reply:
> This is going to create trouble. Compare with a constant and not a string resource. Use string resource just for presentation

Sorry, this comment was missing, when I read.